README.md

```diff
@@ -53,7 +54,13 @@
 
 - [scriptable-calendar-widget](https://github.com/rudotriton/scriptable-calendar-widget) - Customizable calendar.
 
-  <img src="https://raw.githubusercontent.com/rudotriton/scriptable-calendar-widget/main/assets/widget.png" width="400"/>
+  <img src="https://raw.githubusercontent.com/rudotriton/scriptable-calendar-widget/main/assets/scriptable-calendar-widget.jpg" width="400"/>
+
+## Finance
+
+- [CryptoDepotWidget](https://github.com/Martlgap/CryptoDepotWidget) - A Widget showing your personal crypto balances.
+
+  <img src="https://raw.githubusercontent.com/Martlgap/CryptoDepotWidget/main/widget_preview.jpg" width="250"/>
```

Comment:
```suggestion
  <img src="https://raw.githubusercontent.com/Martlgap/CryptoDepotWidget/main/widget_preview.jpg" width="200"/>
```

Do you mind decreasing the image width? I try to have all images in similar widths (200, 400 or 800) for consistency.

Reply:
Oh sure. 200 is fine!